### PR TITLE
Goodbye Jenkins

### PR DESCRIPTION
**Pull request by jaredpar**
This changes the remaining Jenkins badges on the README to point to our
new Azure DevOps badges instead. Also cleans up a few comments on the
switch over PR

### CODE REVIEW

**Comment by jaredpar** [comment]
@dotnet/roslyn-infrastructure for review

File: README.md [hunk 1]
```diff
@@ -5,34 +5,35 @@
...
-**dev16.0-preview2-vs-deps**|[![Build Status](https://ci.dot.net/buildStatus/icon?job=dotnet_roslyn/dev16.0-preview2-vs-deps/windows_debug_vs-integration)](https://ci.dot.net/job/dotnet_roslyn/job/dev16.0-preview2-vs-deps/job/windows_debug_vs-integration/)|[![Build Status](https://ci.dot.net/buildStatus/icon?job=dotnet_roslyn/dev16.0-preview2-vs-deps/windows_release_vs-integration)](https://ci.dot.net/job/dotnet_roslyn/job/dev16.0-preview2-vs-deps/job/windows_release_vs-integration/)|
+**master**|[![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/roslyn/roslyn-integration-CI?label=build&branchname=master&jobname=Windows_VisualStudio_Integration_Tests&configuration=debug)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=15&branchname=master&view=logs)|[![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/roslyn/roslyn-integration-CI?label=build&branchname=master&jobname=Windows_VisualStudio_Integration_Tests&configuration=release)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=15&branchname=master&view=logs)|
```
Comment (sharwell) [inline README.md]: ❓ Is this link correct? When I clicked it, it went to the roslyn-CI queue.

Comment (jaredpar) [reply README.md]: Good catch. That's wrong.

Comment (jaredpar) [reply README.md]: @sharwell fixed

File: README.md [hunk 2]
```diff
@@ -21,10 +21,10 @@
...
+**master-vs-deps**|[![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/roslyn/roslyn-integration-CI?label=build&branchname=master-vs-deps&jobname=Windows_VisualStudio_Integration_Tests&configuration=debug)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=245&branchname=master-vs-deps&view=logs)|[![Build Status](https://dev.azure.com/dnceng/public/_apis/build/status/dotnet/roslyn/roslyn-integration-CI?label=build&branchname=master-vs-deps&jobname=Windows_VisualStudio_Integration_Tests&configuration=release)](https://dev.azure.com/dnceng/public/_build/latest?definitionId=245&branchname=master-vs-deps&view=logs)|
```
Comment (sharwell) [inline README.md]: 😕 Now **master** works for me, but the rest of these are taking me to MachineLearning-CI

File: README.md (hunk 1 above)
Comment (sharwell) [reply README.md]: ➡️ Verified **master** links are fixed.

File: README.md (hunk 2 above)
Comment (jaredpar) [reply README.md]: The links are correct. The problem is we haven't enabled the integration tests in the other branches yet (why they're marked as never run). The ADO behavior for such a link is to take you to the first job definition vs. specified job definition.

**Pull request merged**